main/models.py

```diff
@@ -23,3 +23,26 @@ def __str__(self):
         This is useful if you need to get info only about title of Post object
```

Comment:
This is a lie! If I need a title of post I'll use `Post.title` explicitly. See Zen of Python (PEP 20): **Explicit is better than implicit**.

You should've stated that this is used when rendering model instance as a string.